templates/display.tpl

```diff
@@ -34,7 +34,11 @@
 		</a>
 
 		<a href="{$parentUrl}" class="title">
-			{$title|escape}
+			{if $titleIsHtml}
+				{$title}
```

Comment:
I think we can use the `strip_unsafe_html` here, take a look at how this data is being handled in other templates.